programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+
+    var p_num = 0;
+    var y_num = 0;
```

Comment:
이렇게 값이 변하는 변수는 let으로 쓰는 것이 좋아요
그리고 자바스크립트에서는 _이렇게 snake문법이 아니라 띄어쓰기 부분은 대문자로 쓰는 camel문법을 사용해요
그래서 띄어쓰기 부분은 대문자로 해주셔야해요
```suggestion
    let pCount = 0;
    let yCount = 0;
```